client/src/ui/components/construction/ExistingBuildings.tsx

```diff
@@ -69,14 +70,13 @@ export const ExistingBuildings = () => {
     }),
     [],
   );
-
   const builtBuildings = useEntityQuery([
     Has(Building),
```

Comment:
did you consider adding this `NotValue(Building, { category: "None" })` here instead of `_tmp.filter((building) => building.type != BuildingType.None);` below

Reply:
Makes way more sense indeed! Is this syntax correct (`NotValue(Building, { category: "None" })`) or do I have to apply a pedersen ?